CHANGES_NEXT_RELEASE

```diff
@@ -0,0 +1,2 @@
+- Add: Processing location attributes for EPL (#198)
+
```

Comment:
Maybe the PR should include an EPL example (documented) showing the usage of the new functionality?